### ISSUE TITLE
enhancement request: provide a 5x5 option for 2048

### ISSUE THREAD

**Issue by rcampbel3** [libretro/libretro-2048#25]
2048 is a really great addictive little game. Only one problem -- once you've played for a while 4x4 is pretty easy. I'd really like to have a 5x5 option like this: http://rayxiao92.github.io/2048quintile/

It almost looks like the code has some way to set horizontal and vertical width, we'd just need a way to set GW, GH, and a max target value > 2048. Thanks for the consideration.

 $ 2048
GW:4 GH:4  GSZ:16
SP:8 TSZ:80
BW:360 BH:360 BOY:96 
size:376x464x1504 
[2048] unable to save game data: Permission denied.

**Comment by RobLoach** [comment]
That certainly makes it easier!

**Comment by keithbowes** [comment]
> That certainly makes it easier!

Yes.  GNOME's version of 2048 has 3x3, 4x4, and 5x5 options.  3x3 is the hardest and 5x5 is the easiest.
